src/navigation-bar.tsx

```diff
@@ -581,7 +581,10 @@ export const NavigationBarAction: React.FC<NavigationBarActionProps> = ({childre
     const isInverse = useIsInverseVariant();
     const classes = useNavigationBarActionStyles({isInverse});
     return (
-        <Touchable {...touchableProps} className={classes.touchable}>
+        <Touchable
+            {...touchableProps}
+            className={touchableProps.className ? touchableProps.className : classes.touchable}
+        >
```

Comment:
this should be equivalent, but please, consider my other comment first:
```suggestion
        <Touchable className={classes.touchable} {...touchableProps}>
```